TestHarness/Cucumber/features/ScenarioOutlines.feature

```diff
@@ -67,3 +67,13 @@ Scenario Outline: Deal correctly with backslashes in the examples
   Examples:
     | file path |
     | c:\Temp\  |
+
+
+Scenario Outline: Deal correctly with parenthesis in the examples
+
+  When I have parenthesis in the value, for example a '<overly descriptive field>'
```

Comment:
`.feature` files should be as legible as possible, so to make it grammatically correct it should be "an '<overly".